context/llpcCompiler.cpp

```diff
@@ -3095,6 +3096,9 @@ void Compiler::BuildShaderCacheHash(
         PipelineDumper::UpdateHashForMap(pResUsage->inOutUsage.perPatchBuiltInInputLocMap, &hasher);
         PipelineDumper::UpdateHashForMap(pResUsage->inOutUsage.perPatchBuiltInOutputLocMap, &hasher);
 
+        // Add disassembly pipeline option to hash
+        hasher.Update(pPipelineOptions->includeDisassembly);
```

Comment:
Please add all pipeline options in  if (stageMask & ShaderStageToMask(ShaderStageFragment)) (line 3131). Thanks!